core/db/queries.py

```diff
@@ -1,11 +1,12 @@
 from datetime import datetime
 from typing import Type
 
-from sqlalchemy import UUID, Select, and_, case, or_, select
+from sqlalchemy import UUID, Select, and_, case, desc, func, or_, select
 from sqlalchemy.orm import Query
 
 import core.db.entities as ents
 from core.const import ITLRegion
+from core.db import db
```

Comment:
is this necessary? Since it's used to get sqlalchemy integer, could you just import "Integer" from sqlalchemy above?